include/zephyr/net/wifi_mgmt.h

```diff
@@ -486,6 +486,8 @@ struct wifi_connect_req_params {
 	const uint8_t *key_passwd;
 	/** Private key passwd length */
 	uint8_t key_passwd_length; /* Max 128 */
+	/** Fast BSS Transition used */
+	int ft_used;
```

Comment:
This looks also boolean type, so please use `bool`

Reply:
Updated to `bool`.